### PR TITLE
Updates for admin guide, misc

### LINKED ISSUE #3525: Create Administration guide
## Expected Behavior

There should be a Knative administration guide for cluster administrator tasks.

## Actual Behavior

Tasks are scattered across different Knative topics. There's no central place to go to look for docs on how to set up Knative on a cluster so that it's usable for developers.

## Additional Info

Information architecture (tentative, accepting feedback): https://miro.com/app/board/o9J_lGgYxFs=/

### PR DESCRIPTION
**Pull request by abrennan89**
Fixes #3525 

## Proposed Changes

- Move install, upgrading, logging, metrics docs to admin guide
- CP of #3590
- Updated landing page buttons
- Redirects
- Clean up, editing metrics and logging docs

### CODE REVIEW

**Comment by csantanapr** [comment]
@abrennan89 
Could you add `mkdocs-awesome-pages-plugin>=2.5` to `requirements.txt` that should fix the build

File: docs/admin/install/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ showlandingtoc: "false"
 # Installing Knative
 
 !!! tip
-    You can install a local distribution of Knative for development use, by following the [Getting started guide](../getting-started/getting-started.md){_blank}.
+    You can install a local distribution of Knative for development use by following the [Getting started guide](../getting-started/){_blank}.
```
Comment (omerbensaadon) [inline docs/admin/install/README.md:27]:
```suggestion
    You can install a local distribution of Knative for development use by following the [Getting started guide](../getting-started/). 
```
I think I may have been the one to do this, but internal link = same tab, external link = new tab

@RichardJJG can you add this to the style guide if it's not already there?

File: docs/admin/upgrade/upgrade-installation-with-operator.md [hunk 2]
```diff
@@ -27,24 +20,26 @@ metadata:
...
-If you upgraded Knative Serving and Eventing, enter the following commands to get information about the pods for each namespace:
+You can confirm that your Knative components have upgraded successfully, by viewing the status of the pods for the components in the relevant namespace.
 
-```bash
-kubectl get pods --namespace knative-serving
-```
+!!! note
+    All pods will restart during the upgrade and their age will reset.
+
+### Knative Serving
```
Comment (omerbensaadon) [inline docs/admin/upgrade/upgrade-installation-with-operator.md:34]: This is a situation where I think content tabs would be awesome! 

Github isn't letting me make the recommendation, but let me know if you'd want me to make this change, basically: 

=== "Knative Serving" 
...
=== "Knative Eventing"

File: docs/admin/upgrade/upgrade-installation-with-operator.md [hunk 3]
```diff
@@ -71,31 +76,41 @@ storage-version-migration-eventing-0.20.0-fvgqf   0/1     Completed   0
 sugar-controller-684d5cfdbb-67vsv                 1/1     Running     0          31s
 ```
 
-You can also verify the status of Knative by checking the custom resources:
+<!-- TODO: Make this a snippet for verifying all installations-->
+## Verifying an upgrade by viewing custom resources
+
+You can verify the status of a Knative component by checking that the custom resource `READY` status is `True`.
+
+### Knative Serving
```
Comment (omerbensaadon) [inline docs/admin/upgrade/upgrade-installation-with-operator.md:84]: Content tabs would be cool here too :)

File: docs/admin/upgrade/upgrade-installation-with-operator.md [hunk 4]
```diff
@@ -71,31 +76,41 @@ storage-version-migration-eventing-0.20.0-fvgqf   0/1     Completed   0
...
 ```
+<!--- END snippet-->
 
-## Rollback
+## Rollback to an earlier version
 
-If the upgrade fails, you can rollback to restore your Knative to the previous version. For example, if something goes wrong with an upgrade to 0.22, and your previous version is 0.21, you can apply the following custom resources to restore Knative Serving and Eventing to version 0.21.
+If the upgrade fails, you can rollback to restore your Knative to the previous version. For example, if something goes wrong with an upgrade to 0.23, and your previous version is 0.22, you can apply the following custom resources to restore Knative Serving and Knative Eventing to version 0.22.
 
 For Knative Serving:
```
Comment (omerbensaadon) [inline docs/admin/upgrade/upgrade-installation-with-operator.md:115]: same here

File: docs/admin/upgrade/upgrade-installation.md [hunk 5]
```diff
@@ -64,13 +56,11 @@ For the various subprojects there is a K8s job to help operators perform this mi
...
 
-To upgrade, apply the YAML files for the subsequent minor versions of all
-your installed Knative components and features, remembering to only
-upgrade by one minor version at a time. For a cluster running version 0.20 of the Knative Serving and Eventing components, the following command upgrades the installation to v0.22.0:
+To upgrade, apply the YAML files for the subsequent minor versions of all your installed Knative components and features, remembering to only upgrade by one minor version at a time. For a cluster running version 0.22 of the Knative Serving and Knative Eventing components, the following command upgrades the installation to v0.23.0:
 
 ```bash
-kubectl apply -f https://github.com/knative/serving/releases/download/v0.22.0/serving-core.yaml \
--f https://github.com/knative/eventing/releases/download/v0.22.0/eventing.yaml \
+kubectl apply -f https://github.com/knative/serving/releases/download/v0.23.0/serving-core.yaml \
```
Comment (omerbensaadon) [inline docs/admin/upgrade/upgrade-installation.md:62]:
```suggestion
kubectl apply -f {{ artifact(repo="serving",file="serving-core.yaml")}} \
```
Using Shortcode, I think this is right? See install-serving-with-yaml.md

File: docs/admin/upgrade/upgrade-installation.md [hunk 6]
```diff
@@ -64,13 +56,11 @@ For the various subprojects there is a K8s job to help operators perform this mi
...
+-f https://github.com/knative/eventing/releases/download/v0.23.0/eventing.yaml \
```
Comment (omerbensaadon) [inline docs/admin/upgrade/upgrade-installation.md:63]:
```suggestion
-f {{ artifact(repo="eventing",file="eventing-core.yaml")}} \
```
I'll be sure to put this on TODO to document

File: mkdocs.yml [hunk 7]
```diff
@@ -268,6 +271,26 @@ plugins:
         filename: .index
         collapse_single_pages: true
         strict: false
+    - redirects:
```
Comment (omerbensaadon) [inline mkdocs.yml:274]: neat

**Review by omerbensaadon — COMMENTED** [review]
Left 1 small comment about Content Blocks :) 

Looks great! 

PS: Added a small commit to fix the version check page that I messed up

**Comment by omerbensaadon** [comment]
@googlebot I consent 

CC: @thisisnotapril

**Comment by thisisnotapril** [comment]
Confirmed CLAs; all valid

**Comment by omerbensaadon** [comment]
@googlebot I consent

**Comment by thisisnotapril** [comment]
He said it was OK, Googlebot!

**Pull request merged**